nvtabular/workflow/node.py

```diff
@@ -376,7 +378,7 @@ def _filter_by_type(elements, type_):
     return results
 
 
-def _combine_schemas(elements):
+def _combine_schemas(elements, input_schema=None):
```

Comment:
I think this was also a change for later that wasn't part of the refactor in this PR